### PR TITLE
More Loadout Items Wave 1

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description
The original release of Loadouts to EE did not have much in the way of actual implementation as far as the item list was concerned. This PR seeks to get a head start at fixing that by implementing a fairly sizeable chunk of new items that can be bought with loadout points.

One important addition is the use of "non-clothing-job-related items", which helps further set this version of Loadouts apart from the /tg/ style of loadouts implemented elsewhere. One of the key strengths of this loadout system is that it's not limited to just replacing a job's clothing selections, but can also add items unrelated to clothing. My given example for this is the new option for Security characters(provided they have sufficient number of hours as a Security Officer) can spend roughly half their loadout points to start with a spare magazine for their handgun. 

Secondly, we also now have an opportunity to place non-nanotrasen corporate items in the game, in such a way that players can tie their characters to other corporations. Ostensibly this is to build up to the idea of "Contractor" characters. I hope that this will help broaden people's horizons. 

# On "Syndie" items
I'd like to take this as an opportunity to kind of push people away from the whole thing of "THATS A SYNDICATE ITEM!!!!" wherever possible. For the most part this is very much NRP behavior that I'd like to discourage. With some notable exceptions, this doesn't mean outright antag items should be obtainable through Loadouts. To that end, items such as Interdyne branded smokes, Cybersun branded clothing, and other items with "In-Universe Company Brands" should be moved away from their identity as strictly antag objects, and instead things characters could reasonably have bought with their paycheck. 

# Technical Changes
I had to make some improvements to the way Loadouts handles species, namely it wasn't previously possible to have an item require any entry from a list of species. I updated it so that SpeciesRequirement takes a list of Species, and will accept any entry from that list. My given example for this is the entries for Nitrogen air tanks, which can only be purchased in loadouts by SlimePersons and Vox(who are not currently playable, but I am accounting for them). Other usage of this is in the Oni-specific Security items. I do hope to use this as an opportunity to introduce more species-specific goods in the future. 

# Media

Outerwear
image

More job-related selections available to some departments
image

More variety of trinkets and items, including instruments!
image

Species Specific Items!
image

# Changelog

:cl: VMSolidus
- add: The catalog of items available via Loadouts has been greatly expanded. Have fun customizing your characters!

### CODE REVIEW

**Review by DangerRevolution — APPROVED** [review]

**Comment by VMSolidus** [comment]
image

I have significantly increased the number of existing categories, as well as moved things around, and added more items to fit said categories. I think for a lot of things, any items listed as "exclusive: true" should probably be discounted if they're a job related item that is replacing something you already start with. Such as for example, the option for Security to start with security webbing instead of a belt(I had to add a FILLED webbing just for this purpose). 

Also fun fact, the "Insulated gloves" that are for assistant only, are actually budget insuls. By popular request, assistants can now gamble loadout points on killing themselves with the budget roulette.

**Comment by VMSolidus** [comment]
I've added the Sol Common Translator to loadouts, as a somewhat cheap entry for the items tab. This will serve as a way of simply introducing to players that the translator items exist, and gives an easy tie-in with Languages and Loadouts. 
image

**Comment by VMSolidus** [comment]
How the hell has this not already been merged?

**Comment by DEATHB4DEFEAT** [comment]
uhhhhhhhhhhhhhhhhh

File: Resources/Prototypes/Loadouts/Jobs/engineering.yml [hunk 1]
```diff
@@ -64,19 +64,6 @@
...
-  cost: 3
-  exclusive: true
-  requirements:
-    - !type:CharacterJobRequirement
-      jobs:
-        - AtmosphericTechnician
-  items:
-    - ClothingOuterSuitChicken
-    - ClothingHeadHatChickenhead
-
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Loadouts/Jobs/engineering.yml:79]: Our eggmos suit... @Pspritechologist

File: Resources/Prototypes/Loadouts/categories.yml [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
   id: Outer
 
+- type: loadoutCategory
+  id: Shoes
+
+- type: loadoutCategory
+  id: Species
+
+- type: loadoutCategory
+  id: Uncategorized
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Loadouts/categories.yml:31]: Put this at the top, like the comment says.

File: Resources/Prototypes/Loadouts/uniform.yml [hunk 3]
```diff
@@ -251,11 +367,131 @@
...
+  category: Uniform
+  cost: 3
+  exclusive: true
+  items:
+    - ClothingUniformMNKGymBra
+
+- type: loadout
+  id: LoadoutClothingMNKDressBlack
+  category: Uniform
+  cost: 4 #The legendary black dress
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Loadouts/uniform.yml:470]:
```suggestion
  cost: 4
```

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Comment by VMSolidus** [comment]
@DEATHB4DEFEAT Done.

**Review by Peptide90 — APPROVED** [review]

**Pull request merged**